Overview.bs

```diff
@@ -396,36 +428,49 @@ Offline Usage {#offline-usage}
+of the incremental tech specifiers should be removed.
 
-Patch Based Incremental Transfer {#patch-incxfer}
-=================================================
 
-Font Subset {#font-subset-info}
---------------------------
+Preliminary Definitions {#prelim-definitions}
+=============================================
 
-A <dfn dfn>font subset</dfn> is a modified version of a font file that contains only the data needed to
-render a subset of:
+Font Subset Description {#font-subset-des-def}
+----------------------------------------------
 
-*  the codepoints,
-*  <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">OpenType layout
-    features</a>,
-*  and <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a>.
+A <dfn>font subset description</dfn> specifies the minimum data that a [=font subset=] must possess:
 
-supported by the original font.
-<span class="conform client server" id="conform-font-subset">
-When a subsetted font is used to render text using any combination of the subset codepoints,
-<a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">layout features</a>,
-or <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a> it must render identically to the original font. This includes rendering with
-the use of any optional typographic features that a renderer may choose to use from the original font,
-such as hinting instructions.
-</span>
+*  a set of codepoints,
+*  a set of <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/featuretags">opentype
+    layout features</a>,
+*  and (in the case of a variable font) a <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/otvaroverview#terminology">design-variation space</a>.
+
+Glyph Closure {#glyph-closure-def}
+----------------------------------
 
-A <dfn dfn>font subset definition</dfn> describes the minimum data (codepoints, layout features,
-variation axis space) that a [=font subset=] must possess.
+The <dfn>glyph closure</dfn> of a font subset description is the full set of glyphs in the
+font that may be needed to render any combination of the described codepoints and layout features.
+
+Font Subset {#font-subset-def}
+------------------------------
+
+A <dfn>font subset</dfn> is a version of a font file modified relative to a font subset description
+such that when the font subset is used to render any document or other text string consisting only of
+codepoints, layout features and points in design space included in that description it must render
+identically to the original font.  This includes rendering with the use of any optional typographic
+features that a renderer may choose to use from the original font, such as hinting instructions.
+
+By definition a font subset must contain the glyph closure of the font subset description, although
```

Comment:
Replace "glyph closure" with [=glyph closure=] and it will link back to the glyph closure definition section.